### ISSUE TITLE
Error: timeout of 5000ms exceeded

### ISSUE THREAD

**Issue by wakjoko** [joaostroher/insomnia-plugin-gist-sync#3]
hi @joaostroher ,

thanks for such a great plugin you've made. really loved it but i just got timeout error that's quite disturbing while syncing my long list of endpoints and figured it out by commenting your timeout in the `createGitHubApi` function.

```
function createGitHubApi(token) {
  return axios.create({
    baseURL: 'https://api.github.com/',
    //timeout: 5000,
    headers: { Authorization: `Bearer ${token}` },
  });
}
```

i think this plugin would be super awesome to have for everyone out there.

thanks!

**Comment by joaostroher** [comment]
Hey @wakjoko, thanks. I'am workin on a new version with some improvments. But i will fix it on current version. Thanks

**Comment by joaostroher** [comment]
Fixed on #4 .
